### DISCUSSION TITLE
PurgeCss configuration

### DISCUSSION THREAD

**Discussion by MyNameTony1000** [fancyapps/ui#420] [Q&A]
Hello.
Have you thought about the compatibility of the plugin with purgeCss?
Could you tell me how to correctly register purgeCss exceptions for fancybox?

**Comment by fancyapps** [comment]
Hi,

Sorry, I am not familiar with PurgeCSS, therefore I have no idea what "purgeCss exceptions for fancybox" means.

**Reply by MyNameTony1000** [reply]
Hello.
PurgeCss removes unused css code based on existing classes and attributes in html and js files. Thus, if classes are made up of variables or in other similar cases and are not explicitly written in files, they are deleted. For such cases, there is a field in which parts of classes are written through regular expressions, or the classes themselves are completely that do not need to be deleted.

**Reply by MyNameTony1000** [reply]
And about compatibility, a question arose because I noticed that there are common class names that start with "has- , can-" and do not contain the word "fancybox"
